pkg/kv/kvserver/rangefeed/buffered_sender.go

```diff
@@ -48,38 +50,20 @@ import (
 //               BufferedPerRangeEventSink.Send    BufferedPerRangeEventSink.SendError
 //
 
-// RangefeedSingleBufferedSenderQueueMaxSize is the maximum number of events
-// that the buffered sender will buffer before it starts returning capacity
-// exceeded errors. Updates to this setting are only applied to new
-// MuxRangefeedCalls. Existing streams will use the previous value until
+// RangefeedSingleBufferedSenderQueueMaxPerReg is the maximum number of events
+// that the buffered sender will buffer for a single registration (identified by
+// streamID). Existing MuxRangefeeds will use the previous value until
 // restarted.
 //
-// The default here has been arbitrarily chosen. Ideally,
-//
-//   - We want to avoid capacity exceeded errors that wouldn't have occurred
-//     when the buffered registrations were in use.
-//
-//   - We don't want to drastically increase the amount of queueing allowed for a
-//     single registration.
-//
-// A small buffer may be justified given that:
-//
-//   - One buffered sender is feeding a single gRPC client, so scaling based on
-//     registrations doesn't necessarily make sense. If the consumer is behind, it
-//     is behind.
-//
-//   - Events emitted during catchup scans have their own per-registration buffer
-//     still.
-//
 // TODO(ssd): This is a bit of a stop-gap so that we have a knob to turn if we
 // need to. We probably want each buffered sender (or each consumerID) to be
 // able to hold up to some fraction of the total rangefeed budget. But we are
 // starting here for now.
-var RangefeedSingleBufferedSenderQueueMaxSize = settings.RegisterIntSetting(
+var RangefeedSingleBufferedSenderQueueMaxPerReg = settings.RegisterIntSetting(
 	settings.SystemOnly,
-	"kv.rangefeed.buffered_sender.queue_max_size",
-	"max size of a buffered senders event queue (0 for no max)",
-	kvserverbase.DefaultRangefeedEventCap*8,
+	"kv.rangefeed.buffered_sender.per_registration_max_queue_size",
+	"maximum number of events a single registration can have queued in the event queue (0 for no max)",
+	kvserverbase.DefaultRangefeedEventCap*2,
```

Comment:
Any specific reason we are using DefaultRangefeedEventCap*2 instead of just DefaultRangefeedEventCap?